util/urlutil/redact.go

```diff
@@ -2,16 +2,24 @@ package urlutil
 
 import (
 	"net/url"
+	"os"
 	"regexp"
 	"strings"
 )
 
 const mask = "xxxxx"
 
+func skipRedactCredentials() bool {
+	return os.Getenv("INSECURE_SKIP_REDACT_CREDENTIALS") == "1"
```

Comment:
Just out of an abundance of caution: how hard would it be for, say, a malicious user of some project that uses `earthly` to send a PR that sets this environment variable in that project's CI?

I think probably non-trivial, since users generally don't have a lot of control over the `earthly-buildkitd` instance _even if_ it's a local `buildkitd` ...  but I want to be sure. Open source projects will be reluctant to use `earthly` if insecure options can be set in PRs.

Reply:
i'll remove it, it put it in while debugging some issues.